File: Week-02/day-03/factorio.py
# - Create a function called `factorio`
#   that returns it's input's factorial

def factorio(nr: int):
    factoral = 1
    for i in range(1, nr + 1):
        factoral *= i
    return(factoral)

print(factorio(5))
print(factorio(10))

File: Week-07/Day-03/factorio.js
'use strict';
// - Create a function called `factorio`
//   that returns it's input's factorial

function factorio(number) {
  var result = 1
  for (var i = 1; i <= number; i++) {
    result *= i;
  }
  return result;
}

console.log( factorio(6) );
